product_properties/20-grub.yml

```diff
@@ -0,0 +1,14 @@
+
+{{% if product in ("rhel7", "ol7", "rhel8", "ol8") %}}
+grub2_uefi_boot_path: "/boot/efi/EFI/redhat"
+{{% elif product == "sle12" %}}
+grub2_uefi_boot_path: "/boot/efi/EFI/sles"
+{{% elif "debian" in product %}}
+grub2_boot_path: "/boot/grub"
+{{% elif product <= "ubuntu1804" %}}
+grub2_boot_path: "/boot/grub"
+grub2_uefi_boot_path: "/boot/efi/EFI/ubuntu"
+{{% elif product > "ubuntu1804" %}}
```

Comment:
It seems to be wrong. I have built the RHEL 9 product and I see `grub2_uefi_boot_path: /boot/efi/EFI/ubuntu` in `build/rhel9/product.yml`.

Reply:
Great, and there is more - luckily, the recently-included profile stability test catches them all:
```
Following properties got different values in the alinux2 product:
 - grub2_boot_path from '/boot/grub2' -> '/boot/grub'
 - grub2_uefi_boot_path from '/boot/grub2' -> '/boot/efi/EFI/ubuntu'
...
```